x/stakeibc/keeper/validator_selection.go

```diff
@@ -59,29 +57,40 @@ func (k Keeper) RebalanceDelegationsForHostZone(ctx sdk.Context, chainId string,
 		return errorsmod.Wrapf(err, "unable to get validator deltas for host zone %s", chainId)
 	}
 
-	msgs, rebalacings := k.GetRebalanceICAMessages(hostZone, valDeltaList, numRebalance)
+	msgs, rebalancings := k.GetRebalanceICAMessages(hostZone, valDeltaList)
 
-	// marshall the callback
-	rebalanceCallback := types.RebalanceCallback{
-		HostZoneId:   hostZone.ChainId,
-		Rebalancings: rebalacings,
-	}
-	rebalanceCallbackBz, err := k.MarshalRebalanceCallbackArgs(ctx, rebalanceCallback)
-	if err != nil {
-		return errorsmod.Wrapf(err, "unable to marshal rebalance callback args")
-	}
+	icaBatchSize := 5
```

Comment:
Let's pull this out as a constant. Not sure it makes sense to add it as a full blown module parameter (since we have way to many of those right now), but maybe at least just add it to the top of this file
```go
const IcaBatchSize = 5
```